project1/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Cormorant+Garamond:400,600|Lexend+Mega&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+
+
+
+
+
+    <div class="wrapper">
+    <h1>The Crane Wife</h1>
+    <h2><a href="https://www.theparisreview.org/blog/author/cjhauser/">By CJ Hauser</a> &ensp;  July 16, 2019</h2>
```

Comment:
I think of these two bits of text as separate things -- if I were reading them aloud, I'd have a pause between them -- and, therefore, think they should get their own markup, for example:
```suggestion
    <h2><a href="https://www.theparisreview.org/blog/author/cjhauser/">By CJ Hauser</a></h2>
    <h2>July 16, 2019</h2>
```
This will put them on their own separate lines and we haven't talked yet about how to change that, so, for this assignment, if you don't make this change, it's okay 👍🏻

Reply:
Hi Professor,
I don't know how to make those two elements on a same line, there fore I use &ensp; 
And I already fixed others, thank for your advises, they are so useful!! And I already pushed them, thanks!